engine/sceneManager/components/Component.hpp

```diff
@@ -0,0 +1,29 @@
+﻿//
+// Created by larsv on 12/11/2024.
+//
+
+#ifndef BLOCKYENGINE_COMPONENT_HPP
+#define BLOCKYENGINE_COMPONENT_HPP
+
+#include <memory>
+#include <string>
+
+#include "sceneManager/components/ComponentTransform.hpp"
```

Comment:
wacht components vallen onder de scenemanager, en niet de core?

Reply:
Mogelijke restructure als laatste in deze PR

Reply:
Ik heb hier voor nu alleen de gameobjects en components omhoog gehaald, zodat de include directive een beetje logisch blijft.